src/SDKs/_metadata/cognitiveservices_resource-manager.txt

```diff
@@ -0,0 +1,11 @@
+2017-12-07 03:31:20 UTC
+
+1) azure-rest-api-specs repository information
+GitHub user: felixwa
```

Comment:
@shahabhijeet , I know this is generated by my own fork. Please take a look at this PR:https://github.com/Azure/azure-rest-api-specs/pull/2103. After that PR is approved and merged I will regenerate.